oggm/utils.py

```diff
@@ -2367,7 +2367,8 @@ def copy_to_basedir(self, base_dir, setup='run'):
         else:
             raise ValueError('setup not understood: {}'.format(setup))
 
-    def get_filepath(self, filename, div_id=0, delete=False, filesuffix=''):
+    def get_filepath(self, filename, div_id=0, delete=False,
+                     input_filesuffix='', filesuffix=''):
```

Comment:
In accordance with my comment above, no need for a new keyword argument: ``utils.py`` can be left unchanged